api/graphql/schema.graphql

```diff
@@ -589,8 +589,7 @@ type ApplicantFilter {
   skills: [Skill] @belongsToMany
   # request creation connects to qualifiedClassifications
   qualifiedClassifications: [Classification] @belongsToMany # Filters applicants based on the classifications pools they've qualified in.
-  qualifiedStreams: [LocalizedPoolStream]
-    @rename(attribute: "qualified_streams") # Filters applicants based on the streams of the pools they've qualified in.
+  workStreams: [WorkStream!] @belongsToMany
```

Comment:
Could you add the comment back in?

Reply:
Yup! Added in [add work stream filter comment back in](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12222/commits/d207cfaafe178ac752de295025ae18be80c8bdc7)